src/pcrlock/pcrlock.c

```diff
@@ -5048,6 +5057,7 @@ static int parse_argv(int argc, char *argv[]) {
                 { "help",            no_argument,       NULL, 'h'                 },
                 { "version",         no_argument,       NULL, ARG_VERSION         },
                 { "no-pager",        no_argument,       NULL, ARG_NO_PAGER        },
+                { "full",            no_argument,       NULL, 'f'                 },
```

Comment:
hmm you specify 'f' here as last field, which suggests you also support a short option `-f`. But you don#t really, it neither appears in --help nor  n the 3rd param to getopt_long().

Please make your mind up on that. I think I would avoid the short option for now, simply because -f more often means "--force", hence I#d leave that here, in particular as we actually *do* have a --force switch.

Hence, my suggestion here: change the 'f' to ARG_FULL